internal/output/output.go

```diff
@@ -11,6 +11,14 @@ import (
 	"github.com/ooni/probe-cli/internal/util"
 )
 
+// MeasurementJSON prints the JSON of a measurement
+func MeasurementJSON(j map[string]interface{}) {
```

Comment:
I was about to say "hey, this is not a good name", then I read the package name!